lib/streams.gd

```diff
@@ -1232,3 +1232,21 @@ DeclareGlobalFunction( "UnInstallCharReadHookFunc" );
 ##  <#/GAPDoc>
 ##
 DeclareGlobalFunction( "InputFromUser" );
+
+#############################################################################
+##
+#F  OpenExternal( <filename> )
+##
+##  <#GAPDoc Label="OpenExternal">
+##  <ManSection>
+##  <Func Name="OpenExternal" Arg='filename'/>
+##
+##  <Description>
+##  Open the file <A>filename</A> using the standard application for this file
+##  provided by the operating system. This can be used to open files like HTML and PDF
```

Comment:
```suggestion
##  Open the file <A>filename</A> using the default application for this file
##  in the operating system. This can be used to open files like HTML and PDF
```
I think "default" is a more standard word that "standard" in this context. And in my opinion, someone could interpret "provided by the OS"  to mean that it only uses applications that are _bundled_ with the OS. But maybe not, and even if so, my suggestion is not necessarily better.